service/controller/app/v1/resource/chartoperator/resource.go

```diff
@@ -0,0 +1,196 @@
+package chartoperator
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
+	"github.com/giantswarm/helmclient"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/afero"
+	"gopkg.in/yaml.v2"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/helm/pkg/helm"
+
+	"github.com/giantswarm/app-operator/pkg/tarball"
+	"github.com/giantswarm/app-operator/service/controller/app/v1/key"
+)
+
+const (
+	Name = "chartoperatorv1"
+)
+
+const (
+	chartOperatorNamespace = "giantswarm"
+	chartOperatorRelease   = "chart-operator"
```

Comment:
I would just go with `namespace` and `release` if there is no strong reason for being explicit here. The package scope is already `chartoperator`.